docs/guides/latest/volumesnapshot/statefulset.md

```diff
@@ -0,0 +1,752 @@
+## Snapshot Statefulste's Volumes
```

Comment:
- Normal Backup.
- Normal Restore.
- Advance Use-case:
	- Backup only Single Replica (just BackupConfiguration definition).
	- Restore same data in all replica (just RestoreSession definition).

Stash can also backup only single replica or restore same data on all replicas of a StatefulSet. This is particularly useful when all replicas of the StatefulSet contains same data. For example, in [MongoDB ReplicaSet](https://kubedb.com/docs/0.12.0/guides/mongodb/clustering/replication_concept/) all the pod contains same data. In this case backup only single replica is enough. Similarly, it might be useful in some cases where all the replicas need to be initialize with same data.

This section will show you how you can backup only one replica of a StatefulSet. It will also show how you can initialize all the replicas of a StatefulSet from a single snasphot.